### ISSUE TITLE
Google Sign-in not working/warning in Instagram in-app browser

### ISSUE THREAD

**Issue by y-yeah** [firebase/firebase-tools#3109]
### [REQUIRED] Environment info

**firebase-tools:8.2.0

**Platform: developed on macOS Catalina 10.15.7, bug found on mobile browsers (iPhone/Android)

### [REQUIRED] Test case

Google needs to redirect to complete authentication but Instagram browser does not allow it to happen. Therefore unless a user opens an external browser, google sign-up/in fails and the user is not even warned with the error.

### [REQUIRED] Steps to reproduce

1. Deploy any website using Google sign-up provided by firebase.
2. Send the link of the website to an Instagram user and have him/her open it using the mobile device.
3. Have a user proceed with signing up or signing in using Google account.

### [REQUIRED] Expected behavior

The user should be successfully able to sign up/sign in, or at least be warned that they cannot use Google sign-in inside the app (e.g. inside Facebook messenger it does that).

### [REQUIRED] Actual behavior

The whole screen turns completely white.

**Comment by samtstern** [comment]
@y-yeah I don't think this is an issue with the Firebase CLI. Let me transfer this to the Firebase JS SDK repo.
